doc/releases/release-notes-4.2.rst

```diff
@@ -86,6 +86,10 @@ New APIs and options
     * :c:func:`bt_le_get_local_features`
     * :c:func:`bt_le_bond_exists`
 
+* Stepper
+
+  * :c:func:`stepper_stop()` can be used to stop the stepper motor.
```

Comment:
just the function name, one can read the doc for the details, we are trying to keep this tight, plus the function name is, as it should be, very clear by itself
 
```suggestion
  * :c:func:`stepper_stop()`
```